src/pwquality.conf

```diff
@@ -61,3 +61,15 @@
 #
 # Path to the cracklib dictionaries. Default is to use the cracklib default.
 # dictpath =
+#
+# Prompt user at most N times before returning with error. The default is 1.
+retry = 3
```

Comment:
Please do not set non-default value here.

Reply:
Fixed